infra/functions-python/main.tf

```diff
@@ -267,7 +267,7 @@ resource "google_cloudfunctions2_function" "extract_bb_batch" {
       PUBSUB_TOPIC_NAME = google_pubsub_topic.dataset_updates.name
       PYTHONNODEBUGRANGES = 0
     }
-    available_memory = local.function_extract_bb_config.memory
+    available_memory = "1Gi"
```

Comment:
Why not use the value from the config file?

Reply:
the value in the config file is higher, but the batch function entry point does not require as much memory. I also didn't feel it was necessary to introduce a new variable for this purpose.